setup.py

```diff
@@ -1,6 +1,10 @@
 #!/usr/bin/env python
 
 from distutils.core import setup
+try:
+    import setuptools
```

Comment:
Eventually, we might need setuptools.  But is there really a scenario where it's optional?